nexus/src/external_api/http_entrypoints.rs

```diff
@@ -200,10 +200,10 @@ pub fn external_api() -> NexusApiDescription {
         api.register(vpc_firewall_rules_view)?;
         api.register(vpc_firewall_rules_update)?;
 
-        api.register(rack_list)?;
-        api.register(rack_view)?;
-        api.register(sled_list)?;
-        api.register(sled_view)?;
+        api.register(system_rack_list)?;
+        api.register(system_rack_view)?;
+        api.register(system_sled_list)?;
+        api.register(system_sled_view)?;
```

Comment:
@ahl I'd like your feedback on this specifically. Does the prefixing make sense to you or should we not? If we decide not to when I get to global images we might need to have a one off prefix for that resource to differentiate it.